ztools/ztls/ztls_handshake.go

```diff
@@ -231,6 +239,17 @@ func (m *serverHelloMsg) MakeLog() *ServerHello {
 		sh.ExtendedRandom = make([]byte, len(m.extendedRandom))
 		copy(sh.ExtendedRandom, m.extendedRandom)
 	}
+	if len(m.scts) > 0 {
+		for _, rawSCT := range m.scts {
+			var out parsedAndRawSCT
+			copy(out.Raw, rawSCT)
```

Comment:
You need to allocate space for `Raw` before you copy, otherwise 0 bytes will be copied.